config/locales/en.yml

```diff
@@ -180,6 +180,7 @@ en:
       ?: Unrated — an article that hasn't yet been rated on Wikipedia's quality scale
       does_not_exist: This article does not exist on Wikipedia
     see_more: See more
+    showing_of: 'Showing %{count} of %{total}'
```

Comment:
I suggest a more explanatory name for this interface message, like `articles_shown`. 'showing_of' is likely to cause confusion.